api/v1beta1/glance_webhook.go

```diff
@@ -215,6 +217,26 @@ func (r *Glance) ValidateCreate() (admission.Warnings, error) {
 	glancelog.Info("validate create", "name", r.Name)
 	var allErrs field.ErrorList
 	basePath := field.NewPath("spec")
+
+	for key, glanceAPI := range r.Spec.GlanceAPIs {
```

Comment:
I first had it in ValidateCreate for the GlanceSpecCore but then we'd have to pass down the glance name. I have no real preference, except that it we call the GlanceSpecCore from openstack-operator webhook and changing the params of the func will result in some CI dep madness.